rmw_iceoryx_cpp/src/internal/iceoryx_name_conversion.cpp

```diff
@@ -93,20 +69,56 @@ get_name_n_type_from_iceoryx_service_description(
   const std::string & instance,
   const std::string & event)
 {
-  std::string topic_name;
-  std::string type_name;
-
-  if (service.find("/msg/") != std::string::npos) {
+  if (event == "data") {
```

Comment:
Where is "data" coming from?

Reply:
iceoryx uses three strings to identify a specific topic instance. We only use two in the rmw_iceoryx. 'data' is more or less an arbitrary name for the not used parameter in the iceoryx service description.

Reply:
I guess it would be good to put it into a `static const char[]`. I will add a change.